tokio-console/src/config.rs

```diff
@@ -98,6 +116,26 @@ pub struct Config {
     pub subcmd: Option<OptionalCmd>,
 }
 
+/// Known warnings that can be enabled or disabled.
+#[derive(clap::ValueEnum, Clone, Debug, Deserialize, Serialize, PartialEq, Eq)]
+pub(crate) enum KnownWarnings {
+    SelfWakePercent,
```

Comment:
I feel that the "percent" bit here isn't necessary (I can't think of another way we could lint self wakes). Let's rename this `SelfWakes`.

```suggestion
    SelfWakes,
```

Reply:
Fixed in https://github.com/tokio-rs/console/pull/493/commits/3c41d0d70201d2d01a4a5a3ac037faf18a23a3a7